ginicapture/src/doc/source/integration.rst

```diff
@@ -0,0 +1,528 @@
+   This interface is used to upload, analyze and delete documents. See the `reference documentation
+   <http://developer.gini.net/gini-capture-sdk-android/ginicapture/dokka/ginicapture/net.gini.android.capture.network/-gini-capture-network-service/index.html>`_
+   for details.
+
+* ``GiniCaptureNetworkApi``
+   This interface is used to declare network tasks which should be called by you outside of the Gini Capture SDK (e.g.,
+   for sending feedback after the user saw and potentielly corrected the extractions).  See the `reference documentation
+   <http://developer.gini.net/gini-capture-sdk-android/ginicapture/dokka/ginicapture/net.gini.android.capture.network/-gini-capture-network-api/index.html>`_
+   for details.
+
+Default Implementation
+~~~~~~~~~~~~~~~~~~~~~~
+
+The quickest way to add networking is to use the `Gini Capture Network
+Library <https://github.com/gini/gini-capture-sdk-android/tree/master/ginicapture-network>`_.
+
+To use it add the ``gini-capture-network-lib`` dependency to your app's ``build.gradle``:
+
+.. code-block:: groovy
+
+    dependencies {
+        ...
+        implementation 'net.gini:gini-capture-network-lib:1.3.1'
+    }
+
+For the Gini Capture SDK to be aware of the default implementations create the instances and pass
+them to the builder of ``GiniCapture``:
+
+.. code-block:: java
+
+    GiniCaptureDefaultNetworkService networkService = 
+        GiniCaptureDefaultNetworkService.builder((Context) this)
+            .setClientCredentials(myClientId, myClientSecret, myEmailDomain)
+            .build();
+
+    GiniCaptureDefaultNetworkApi networkApi = 
+        GiniCaptureDefaultNetworkApi.builder()
+            .withGiniCaptureDefaultNetworkService(networkService)
+            .build();
+
+    GiniCapture.newInstance()
+        .setGiniCaptureNetworkService(networkService)
+        .setGiniCaptureNetworkApi(networkApi)
+        .build();
+
+Both default implementations follow the builder pattern. See the documentation of
+`GiniCaptureDefaultNetworkService.Builder
+<http://developer.gini.net/gini-capture-sdk-android/network/javadoc/net/gini/android/capture/network/GiniCaptureDefaultNetworkService.Builder.html>`_
+and `GiniCaptureDefaultNetworkApi.Builder
+<http://developer.gini.net/gini-capture-sdk-android/network/javadoc/net/gini/android/capture/network/GiniCaptureDefaultNetworkApi.Builder.html>`_
+for configuration options.
+
+Custom Implementation
+~~~~~~~~~~~~~~~~~~~~~
+
+You can also provide your own networking by implementing the ``GiniCaptureNetworkService`` and the
+``GiniCaptureNetworkApi`` interfaces. Pass your instances to the builder of ``GiniCapture`` as shown
+above.
+
+You may also use the `Gini Pay API Library <https://github.com/gini/gini-pay-api-lib-android>`_ for Android or implement
+communication with the Gini API yourself.
+
+Sending Feedback
+~~~~~~~~~~~~~~~~
+
+You can help improve our service if your app sends feedback for the extractions the Gini Pay API delivered. Your app
```

Comment:
I wonder if we should keep this line in our SDK because of "can". According to the contracts they should send us feedback, right? Maybe remove this sentence entirely and start with "Your app should.."?